heudiconv/utils.py

```diff
@@ -46,7 +46,8 @@
     'patient_sex',           # 23
     'date',                  # 24
     'series_uid',            # 25
- ]
+    'time',                  # 26
+]
```

Comment:
this would conflict with WiP of #637 / #581 but the main concern is that I do not see it needed/used at all by the target need of this PR -- conversion of physio data. Spurious wishful change? ;-)